Airbnb.Infrastructure/Repositories/ApartmentRepository.cs

```diff
@@ -15,9 +15,17 @@ public ApartmentRepository(AppDbContext context)
         _context = context;
     }
 
-    public async Task<List<Apartment>> GetAllAsync()
+    public async Task<List<Apartment>> GetAsync(int pageNumber, int pageSize, string? location = null)
```

Comment:
Create page response for it, like currentPage, totalCount, hasNextPage(bool), etc